### PR TITLE
#64: [Popover] - Statelessness (closes #64)

### LINKED ISSUE #64: [Popover] - Statelessness
## REQS
- Popover should be able to act as a stateless component
- it should accept an `isOpen` prop or something

### PR DESCRIPTION
**Pull request by FrancescoCioria**
Issue #64

### CODE REVIEW

**Comment by FrancescoCioria** [comment]
NOTE: first two commits are from #67 (they will disappear after merge + rebase)

**Comment by FrancescoCioria** [comment]
@giogonzo  ready to be reviewed

File: src/popover/Popover.js [hunk 1]
```diff
@@ -174,43 +172,70 @@ const Popover = React.createClass({
     }
   },
 
-  onPopoverStateChange() {
-    const { onShow, onHide } = this.getPopoverProps();
-    const isAbsolute = this.isAbsolute();
-    if (this.state.isOpen) {
-      onShow();
-      if (isAbsolute) {
+  isStatefull(props) {
```
Comment (giogonzo) [inline src/popover/Popover.js]: s/statefull/stateful :)

File: src/popover/Popover.js [hunk 2]
```diff
@@ -174,43 +172,70 @@ const Popover = React.createClass({
...
   togglePopover() {
-    this.setState(
-      {isOpen: !this.state.isOpen},
-      this.onPopoverStateChange
-    );
+    this.setIsOpen(!this.isOpen());
+  },
+
+  setIsOpen(isOpen) {
+    console.log(isOpen);
```
Comment (giogonzo) [inline src/popover/Popover.js]: leftover

**Comment by giogonzo** [comment]
:+1: except for minor stuff

File: src/popover/Popover.js (hunk 2 above)
Comment (FrancescoCioria) [inline src/popover/Popover.js]: already fixed in #69

File: src/popover/Popover.js (hunk 1 above)
Comment (FrancescoCioria) [inline src/popover/Popover.js]: fixed now in #69 with last commit (https://github.com/buildo/react-components/commit/ec77641a80bdfc03a5964ea3a0f0d86470df74ce)

**Pull request merged**